File: algorithms/Strings/Easy/ExcelSheetColumnNumber/VSDExcelSheetColumnNumber.py

def excelSheetColumnTitleToNumber(string):
    if(len(string)==0):
            print("Empty String")
            return 
    sum=0
    for i in range(len(string)):
          sum = (sum*26 + ord(string[i])-64)
    return sum


string="AA"
print(excelSheetColumnTitleToNumber(string))

File: algorithms/Strings/Easy/ExcelSheetColumnNumber/VSDExcelSheetColumnNumber.java

public class Main{

    static void colNo(String s){
        int len=s.length();
        if(len==0){
            System.out.println("Empty String");
            return;
        }
        int sum=0;
        for(int i=0;i<len;i++){
            sum=(sum*26+((int)s.charAt(i)-64));

        }
        System.out.println(sum);
    }
    public static void main(String[] args) {
        String s="AA";
        
    }
}